website/docs/style/icon/icon-a11y.md

```diff
@@ -0,0 +1,43 @@
+---
+title: A11y
+a11y: AA
+---
+
+@## What component has
+
+### Roles & attributes
+
+The list below describes roles and attributes that component already has.
+
+| Role  | Attribute            | Element | Usage                                                                                                                        |
```

Comment:
There is no role="img" and such atributes in the component yet.

You can read https://www.w3.org/TR/SVG-access/ and https://css-tricks.com/accessible-svgs/ to understand what we can add and what our developers should add.

Reply:
And always check how component is read by screenreader. It usually helps to find all inconsistencies.

Reply:
Updated guide for Icon component.